rkyv_derive/src/archive/struct.rs

```diff
@@ -422,7 +422,7 @@ fn generate_partial_ord_impl(
                 #(
                     match other.#members.partial_cmp(&self.#members) {
                         Some(::core::cmp::Ordering::Equal) => (),
-                        x => return x,
+                        x => return x.map(|o| o.reverse()),
```

Comment:
```suggestion
                        x => return x.map(::core::cmp::Ordering::reverse),
```
I just prefer to avoid writing closures when I can.